### ISSUE TITLE
[aws][iam] InvalidClientTokenId => The security token included in the request is invalid

### ISSUE THREAD

**Issue by drnic** [fog/fog#3387]
Using fog/1.26.0 fog-core/1.27.2

```
>> iam = Fog::AWS::IAM.new(Fog.credentials)
#<Fog::AWS::IAM::Real:70166177964140 @use_iam_profile=nil @connection_options={:debug_response=>true, :headers=>{"User-Agent"=>"fog/1.26.0 fog-core/1.27.2"}, :persistent=>false} @instrumentor=nil @instrumentor_name="fog.aws.iam" @host="iam.amazonaws.com" @path="/" @persistent=false @port=443 @scheme="https" @connection=#<Fog::XML::Connection:0x007fa1ac39dd48 @excon=#<Excon::Connection:7fa1ac397bc8 @data={:chunk_size=>1048576, :ciphers=>"HIGH:!SSLv2:!aNULL:!eNULL:!3DES", :connect_timeout=>60, :debug_request=>false, :debug_response=>true, :headers=>{"User-Agent"=>"fog/1.26.0 fog-core/1.27.2"}, :idempotent=>false, :instrumentor_name=>"excon", :middlewares=>[Excon::Middleware::ResponseParser, Excon::Middleware::Expects, Excon::Middleware::Idempotent, Excon::Middleware::Instrumentor, Excon::Middleware::Mock], :mock=>false, :nonblock=>true, :omit_default_port=>false, :persistent=>false, :read_timeout=>60, :retry_limit=>4, :ssl_verify_peer=>true, :tcp_nodelay=>false, :thread_safe_sockets=>true, :uri_parser=>URI, :versions=>"excon/0.42.1 (x86_64-darwin14.0) ruby/2.1.5", :write_timeout=>60, :host=>"iam.amazonaws.com", :path=>"/", :port=>443, :query=>nil, :scheme=>"https"} @socket_key="https://iam.amazonaws.com:443">> @aws_session_token=nil @aws_credentials_expire_at=nil @signer=#<Fog::AWS::SignatureV4:0x007fa1ac397790 @region="us-east-1", @service="iam", @aws_access_key_id="XXX", @hmac=#<Fog::HMAC:0x007fa1ac3976f0 @key="XXX", @digest=#<OpenSSL::Digest: XXX>, @signer=#<Proc:0x007fa1ac397650@/Users/drnic/.rvm/gems/ruby-2.1.5/gems/fog-core-1.27.2/lib/fog/core/hmac.rb:28 (lambda)>>>>

>> iam.list_account_aliases
Fog::AWS::IAM::Error: InvalidClientTokenId => The security token included in the request is invalid.
	from /Users/drnic/.rvm/gems/ruby-2.1.5/gems/excon-0.42.1/lib/excon/middlewares/expects.rb:6:in `response_call'
	from /Users/drnic/.rvm/gems/ruby-2.1.5/gems/excon-0.42.1/lib/excon/middlewares/response_parser.rb:8:in `response_call'
```

**Comment by fcheung** [comment]
What type of credentials are they (e.g. are they federated credentials etc) ?

**Comment by plribeiro3000** [comment]
Closed in favor of fog/fog-aws#21
